src/lib/remark-plugins/rewrite-static-tutorials-assets/index.ts

```diff
@@ -50,7 +60,20 @@ export const rewriteStaticAssetsPlugin: Plugin = () => {
 			const isVercelBuild =
 				process.env.VERCEL_ENV === 'production' ||
 				process.env.VERCEL_ENV === 'preview'
+
 			const newUrl = new URL(ASSET_API_ENDPOINT)
+			const { hash, pathname } = new URL(
+				node.url,
+				'https://developer.hashicorp.com'
```

Comment:
(question) Is `'https://developer.hashicorp.com'` mostly an arbitrary arg to satisfy the `URL` constructor?

If so, a quick comment highlighting that would be a nice breadcrumb that future devs will appreciate

Reply:
ah yes it is, will add 🙇🏻